boot/modeenv.go

```diff
@@ -325,6 +328,54 @@ func (m *Modeenv) WriteTo(rootdir string) error {
 	return nil
 }
 
+// dummy to verify interface match
+var _ secboot.ModelForSealing = (*modeenvModelForSealing)(nil)
+
+// modeenvModelForSealing is a helper type that implements
+// github.com/snapcore/secboot.SnapModel interface.
+type modeenvModelForSealing struct {
+	brandID        string
+	model          string
+	grade          asserts.ModelGrade
+	modelSignKeyID string
+}
+
+func (m *modeenvModelForSealing) BrandID() string           { return m.brandID }
+func (m *modeenvModelForSealing) SignKeyID() string         { return m.modelSignKeyID }
+func (m *modeenvModelForSealing) Model() string             { return m.model }
+func (m *modeenvModelForSealing) Grade() asserts.ModelGrade { return m.grade }
+func (m *modeenvModelForSealing) Series() string            { return release.Series }
```

Comment:
Series() doesn't seem to be used/reached by tests in the package

Reply:
added a check